include/util/coordinate.hpp

```diff
@@ -151,6 +151,7 @@ struct Coordinate
     bool IsValid() const;
     friend bool operator==(const Coordinate lhs, const Coordinate rhs);
     friend bool operator!=(const Coordinate lhs, const Coordinate rhs);
+    friend bool operator<(const Coordinate lhs, const Coordinate rhs);
```

Comment:
This is technically a left-over, since I had some sets in there before. Removing again.